doc/source/reference/storage/cinder-guide-lightbits.rst

```diff
@@ -0,0 +1,33 @@
+.. cinder-guide-Lightbits:
+
+=====================================
+Lightbits labs storage for OpenStack
```

Comment:
I would just drop the "labs" here. If you want to keep it, it needs to be capitalized, as the name is "Lightbits Labs", not "Lightbits labs".